### PR TITLE
CW-208 : Landing Page Sponsor

### PR DESCRIPTION
**Pull request by paulatennent**
# Change

- Updated landing page sponsor section to match current design include high fidelity
- Cleaned up code to match style guide and make use of v-rows/cols

## Change reason

- to update to match the current design

### CODE REVIEW

File: frontend/src/styles/components/_buttons.scss [hunk 1]
```diff
@@ -19,7 +18,7 @@ $btn-radius: 5px;
 
 // ----------------------------------------------------------------------
 .btn {
-  display: inline-flex;
+  //display: inline-flex;
```
Comment (glebme) [inline frontend/src/styles/components/_buttons.scss:21]: why was this changed?

File: frontend/src/views/Home.vue [hunk 2]
```diff
@@ -260,26 +286,48 @@ export default {
...
+  background-image: url(../assets/supportusbackground.png);
+  background-size: cover;
+  background-position: 20% 50%;
 }
 
-#sponsor p {
-  font-size: 30px;
-  padding: 0 0 25px 0;
+.sponsor--h1 {
+  @extend h1;
```
Comment (glebme) [inline frontend/src/views/Home.vue:298]: you don't need to extend for these because they are styles associated with the tag itself. So `<h1>` tag already comes with h1 styles.

File: frontend/src/views/Home.vue [hunk 3]
```diff
@@ -80,35 +81,58 @@
...
+          <!-- network image -->
+          <v-col cols="4" align="center" justify="center" class="pa-0">
+            <img src="@/assets/share.svg" class="sponsor--img">
+          </v-col>
+          <!-- h4 and p text adjacent to network image -->
+          <v-col cols="8" class="pa-0">
+            <h4 class=sponsor--h4>Interact With Our Huge Active Community</h4>
+            <p class=sponsor--p> We have an extremely active community of ~3000 CSE students,
+              achieved through our offering of career, social and educational events. </p>
+          </v-col>
```
Comment (glebme) [inline frontend/src/views/Home.vue:124]: _nit_: This can be done in a for loop for ease of use but I can refactor that. For extra guidance, look at Jie's footer and how he has taken the data away from the html and used vue's for loop to render the data.

**Review by glebme — APPROVED** [review]
If you could get back to me with answers for the questions I had below.

File: frontend/src/views/Home.vue (hunk 2 above)
Comment (glebme) [reply frontend/src/views/Home.vue:298]: So if you could remove all occurrences of this type of extend.

File: frontend/src/styles/components/_buttons.scss (hunk 1 above)
Comment (paulatennent) [reply frontend/src/styles/components/_buttons.scss]: it removed the hyper link underline, i just put it back and fixed it in a better way
